spec/sidekiq/lighthouse/document_upload_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+require 'lighthouse/document_upload'
+
+RSpec.describe Lighthouse::DocumentUpload, type: :job do
+  subject { described_class }
+
+  let(:notify_client_stub) { instance_double('VaNotify::Service') }
```

Comment:
Please switch this back to use the constant and remove the file from the `rubocop_todo.yml`

`let(:notify_client_stub) { instance_double(VaNotify::Service) }`

Reply:
When I use the constant I get an error saying that it is not initialized, do you know how to avoid that?